packages/integration-tests/index.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const uuidv4 = require('uuid/v4');
+const fs = require('fs-extra');
+const { s3, sfn } = require('@cumulus/common/aws');
+
+const executionStatusNumRetries = 20;
+const waitPeriodMs = 5000;
+
+/**
+ * Wait for the defined number of milliseconds
+ *
+ * @param {integer} waitPeriod - number of milliseconds to wait
+ * @returns {Promise.<undefined>} - promise resolves after a given time period
+ */
+function timeout(waitPeriod) {
+  return new Promise((resolve) => setTimeout(resolve, waitPeriod));
+}
+
+/**
+ * Get the list of workflows for the stack
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @returns {Object} list as a JSON object
+ */
+async function getWorkflowList(stackName, bucketName) {
+  const key = `${stackName}/workflows/list.json`;
+
+  return s3().getObject({ Bucket: bucketName, Key: key }).promise()
+    .then((workflowJson) => JSON.parse(workflowJson.Body.toString()));
+}
+
+/**
+ * Get the template JSON from S3 for the workflow
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {string} workflowName - workflow name
+ * @returns {Object} template as a JSON object
+ */
+async function getWorkflowTemplate(stackName, bucketName, workflowName) {
+  const key = `${stackName}/workflows/${workflowName}.json`;
+  return s3().getObject({ Bucket: bucketName, Key: key }).promise()
+    .then((templateJson) => JSON.parse(templateJson.Body.toString()));
+}
+
+/**
+ * Get the workflow ARN for the given workflow from the
+ * template stored on S3
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {string} workflowName - workflow name
+ * @returns {string} - workflow arn
+ */
+async function getWorkflowArn(stackName, bucketName, workflowName) {
```

Comment:
This is not an async function.  `await` is never used.

Reply:
Fixed.